app/migrations/Version20161026101817.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Sylius\Component\Customer\Model\CustomerGroupInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20161026101817 extends AbstractMigration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_customer_group ADD code VARCHAR(255) NOT NULL');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_7FCF9B0577153098 ON sylius_customer_group (code)');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function down(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP INDEX UNIQ_7FCF9B0577153098 ON sylius_customer_group');
+        $this->addSql('ALTER TABLE sylius_customer_group DROP code');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function postUp(Schema $schema)
+    {
+        $this->addSql('UPDATE sylius_customer_group SET sylius_customer_group.code = UCASE (sylius_customer_group.name)');
```

Comment:
This won't work since `ALTER TABLE` in `up()` will complain that code is already null.